client/src/main/scala/org/http4s/armeria/client/ArmeriaClient.scala

```diff
@@ -100,7 +109,10 @@ private[armeria] final class ArmeriaClient[F[_]] private[client] (
           .body()
           .toStreamBuffered[F](1)
           .flatMap(x => Stream.chunk(Chunk.array(x.array())))
-    } yield Response(status = status, headers = toHeaders(headers), body = body)
+    } yield Response(
+      status = status,
+      headers = toHeaders(headers),
+      entity = Entity.Default(body, None))
```

Comment:
Ditto, should we set the length of the content?

Reply:
Here I think we can propagate the content length from the underlying response. It just wouldn't be used on the `http4s` side, but it can be used somehow by users :+1: